sample/index.html

```diff
@@ -50,15 +53,17 @@
 		will be working on in the next few iterations of the project. Stay
 		tuned for some updates soon!</p>
 </div>
+
 <script src="../build/ckeditor.js"></script>
 <script>
 	ClassicEditor.create( document.querySelector( '#editor' ) )
-		.then( editor => {
-			window.editor = editor;
-		} )
-		.catch( err => {
-			console.error( err.stack );
-		} );
+	.then( editor => {
```

Comment:
Is that a rule? I was pretty sure that if the code before a method is one-line long, then we're starting the next lines with indentation.